cmd/program/program.go

```diff
@@ -196,16 +272,22 @@ func (p *Project) CreateMainFile() error {
 		return err
 	}
 
-	err = p.CreateFileWithInjection(internalServerPath, projectPath, "server.go", "server")
+	if p.DBDriver != "none" {
+		err = p.CreateFileWithInjection(internalServerPath, projectPath, "routes.go", "routesWithDB")
+		err = p.CreateFileWithInjection(internalServerPath, projectPath, "server.go", "serverWithDB")
+	} else {
+		err = p.CreateFileWithInjection(internalServerPath, projectPath, "routes.go", "routes")
+		err = p.CreateFileWithInjection(internalServerPath, projectPath, "server.go", "server")
```

Comment:
i think you should error every time

Reply:
Yeah, I'm not totally convinced with the whole approach. I'm probably going to change it to a more appropriate way.